curriculum/challenges/english/20-upcoming-python/learn-classes-and-objects-by-building-a-sudoku-solver/6569e37ec28e853628f18a86.md

```diff
@@ -0,0 +1,99 @@
+---
+id: 6569e37ec28e853628f18a86
+title: Step 57
+challengeType: 20
+dashedName: step-57
+---
+
+# --description--
+
+Add all the variables to an iterable and pass it in the `all` function.
+`all` would make sure that all of the validation results are True.
+
+Recall that the syntax is as follows:
+    
+```js
+all([variable1, variable2, variable3])
+```
```

Comment:
```suggestion
Insert `valid_in_row`, `valid_in_col`, and `valid_in_square` into a list and pass it to the `all()` function. This will verify that all the function calls return `True`.
```

`all` has been already used several times.